source/assets/critical/c_landingpage.scss

```diff
@@ -15,81 +15,42 @@ body.body--loading.body--loading {
   }
 
   .collection {
-    margin: 0 0 $collection-gap;
+    max-width: 100%;
+    margin: $collection-gap 0 0;
 
     @include tablet-up {
-      margin-bottom: $collection-gap-tablet-up;
+      margin-top: $collection-gap-tablet-up;
     }
 
     @include desktop-wide {
-      margin-bottom: $collection-gap-desktop-wide;
+      margin-top: $collection-gap-desktop-wide;
     }
 
-    max-width: 100%;
-
     .collection__content-wrapper {
       max-width: $desktop-wide-size;
       margin: 0 auto;
       background-color: transparent;
       box-shadow: none;
-
-      @include tablet-up {
-      margin-top: 20px;
-      }
     }
   }
 
   .collection--branded {
+    background-size: cover;
+
     .collection__branding-wrapper {
```

Comment:
das steht glaubs schon genau gleich in collections.scss (oder wird das von irgendwo überschrieben?)

Reply:
Ne das muss so. In collections.scss dient es dazu, dass die Komponente selbstständig "funktioniert"/korrekt aussieht. Damit's nicht herumspringt, braucht's einen Teil davon in diesem Critical CSS. Im collections.scss wollen wir es nicht hinausnehmen, da sonst beim Verwenden dieser Komponente an einem anderen Ort als auf einer Landingpage nichts mehr richtig aussehen würde.